src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PersonControllerTest.kt

```diff
@@ -300,24 +310,28 @@ internal class PersonControllerTest(
           result.response.contentAsString.shouldBe(
             """
         {
-        "data": {
-          "firstName": "Silly",
-          "lastName": "Sobbers",
-          "middleName": null,
-          "dateOfBirth": null,
-          "gender": null,
-          "ethnicity": null,
-          "aliases": [],
-          "identifiers": {
-              "nomisNumber": null,
-              "croNumber": null,
-              "deliusCrn": null
+          "data": {
+            "probationOffenderSearch": {
+              "firstName": "Silly",
+              "lastName": "Sobbers",
+              "middleName": null,
+              "dateOfBirth": null,
+              "gender": null,
+              "ethnicity": null,
+              "aliases": [],
+              "identifiers": {
+                "nomisNumber": null,
+                "croNumber": null,
+                "deliusCrn": null
+              },
+            "pncId": null,
+            "hmppsId": null,
+            "contactDetails": null
           },
-          "pncId": null,
-          "hmppsId": null,
-          "contactDetails": null
-        }
+          "prisonerOffenderSearch": null
```

Comment:
Can we have a test to verify prisonerOffenderSearch data is populated please?

Reply:
okay!